flask_app/db.py

```diff
@@ -15,7 +15,7 @@ def _create_engine():
     database = config.pop('database')
 
     return sqlalchemy.create_engine(
-        f"mysql+mysqldb://{username}:{password}@{host}:3306/{database}",
+        f"mysql+pymysql://{username}:{password}@{host}:3306/{database}",
```

Comment:
Ran into some issues locally with mysqldb, but it seems like `pymysql` works consistently, so I'll use that one for now.

Reply:
Hi @AndrewRadev 
This is just fine. 
Just fyi as it is now completely irrelevant, we had observed that `pymysql` was necessary for the streamlit cloud app to work.